<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> ADAMS KIT V1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
    color: #b1b1b1;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

QPushButton:hover
{
    
}

QPushButton:focus {
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QRadioButton:focus {
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QSlider:focus {
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="settingsPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>551</width>
          <height>401</height>
         </rect>
        </property>
        <widget class="QWidget" name="generalPage">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>360</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="saveGeneral">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>SAVE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="backGeneral">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>BACK</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>160</y>
            <width>401</width>
            <height>55</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>WARNING TYPE</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="VISUAL">
               <property name="text">
                <string>VISUAL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="AUDIENCE">
               <property name="text">
                <string>AUDIENCE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="VISUALAUDIENCE">
               <property name="text">
                <string>VISUAL / AUDIENCE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>367</width>
            <height>55</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>DETECTION TYPE</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="OBJECTS">
               <property name="text">
                <string>OBJECTS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LANE">
               <property name="text">
                <string>LANE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="OBJECTLANE">
               <property name="text">
                <string>OBJECT / LANE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>261</width>
            <height>55</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>VIEW TYPE</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="REAL">
               <property name="text">
                <string>REAL VIEW</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="ANIMATED">
               <property name="text">
                <string>ANIMATED VIEW</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>240</y>
            <width>241</width>
            <height>55</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>LANGUAGE</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QRadioButton" name="ENGLISH">
               <property name="text">
                <string>ENGLISH</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="BANGLA">
               <property name="text">
                <string>BANGLA</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="advancedPage">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>411</width>
            <height>184</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>OBJECT DETECTION</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="objectSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>LANE DETECTION</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="LaneSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>VOLUME</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="volumeSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>360</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="saveAdvanced">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>SAVE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="backAdvanced">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>BACK</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="logPage">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>60</y>
            <width>127</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QPushButton" name="motionButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>MOTION</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gpsButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>GPS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>VIDEO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="formatButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>FORMAT SD CARD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="unmountButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>UNMOUNT SD CARD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logBackButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>BACK</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="profilePage">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>401</width>
            <height>62</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="horizontalSpacing">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>PERFORMANCE INDEX</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>LICENSE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="generalButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>GENERAL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="advancedButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ADVANCED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>LOG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profileButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>PROFILE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>BACK</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="onPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="onButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>TURN ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>SETTINGS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>onButton</tabstop>
  <tabstop>settingsButton</tabstop>
  <tabstop>generalButton</tabstop>
  <tabstop>advancedButton</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>profileButton</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>REAL</tabstop>
  <tabstop>ANIMATED</tabstop>
  <tabstop>OBJECTS</tabstop>
  <tabstop>LANE</tabstop>
  <tabstop>OBJECTLANE</tabstop>
  <tabstop>VISUAL</tabstop>
  <tabstop>AUDIENCE</tabstop>
  <tabstop>VISUALAUDIENCE</tabstop>
  <tabstop>ENGLISH</tabstop>
  <tabstop>BANGLA</tabstop>
  <tabstop>saveGeneral</tabstop>
  <tabstop>backGeneral</tabstop>
  <tabstop>objectSlider</tabstop>
  <tabstop>LaneSlider</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>saveAdvanced</tabstop>
  <tabstop>backAdvanced</tabstop>
  <tabstop>motionButton</tabstop>
  <tabstop>gpsButton</tabstop>
  <tabstop>videoButton</tabstop>
  <tabstop>formatButton</tabstop>
  <tabstop>unmountButton</tabstop>
  <tabstop>logBackButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
